tests/Security/yaml_fixtures/expected_authenticator/simple_security_with_added_access_control.yaml

```diff
@@ -0,0 +1,16 @@
+security:
+    # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
+    providers:
+        in_memory: { memory: ~ }
+
+    firewalls:
+        dev: ~
+        main:
+            anonymous: true
+            guard:
+                authenticators:
+                    - App\Security\AppCustomAuthenticator
+    access_control:
+        -
+            path: ^/login$
+            roles: IS_AUTHENTICATED_ANONYMOUSLY
```

Comment:
Is there any way to use the inline syntax (as in `simple_security_with_access_control.yaml`) instead of that one?

Reply:
Not that I'm aware of. It was pointed out in the [related issue](https://github.com/symfony/maker-bundle/issues/396#issuecomment-496578320) that this might happen

Reply:
Ok, thank you for the clarification!